src/pivot-table/constants.ts

```diff
@@ -1,5 +1,5 @@
 export const DEFAULT_FONT_SIZE = "12px";
-export const DEFAULT_FONT_FAMILY = "Source Sans Pro, Arial, sans-serif";
+export const DEFAULT_FONT_FAMILY = "Source Sans Pro, sans-serif";
```

Comment:
why removing Arial?

Reply:
is it because of this: https://github.com/qlik-trial/qlik-chart-modules/blob/d6c824406114ced8dd7b6f113e1b1f773f9451e6/src/resources/fonts/getAvailableFonts.js#L5C18-L5C18?

Reply:
That and the Sense themes use `"Source Sans Pro, sans-serif"`